### PR TITLE
fix(time): tooltip formatter for time axis

### PR DESCRIPTION
**Pull request by Ovilia**
## Brief Information

This pull request is in the type of:

- [x] bug fixing
- [ ] new feature
- [ ] others

### What does this PR do?

Fix default formatter for tooltip with time axis. Relates to #12947 

### Before: What was the problem?

Time axis returned `'day'` with tooltip, and the `'d'` was replaced by date of month and got something like `'2ay'`.

### After: How is it fixed in this PR?

Default time formatter in tooltip is `'{yyyy}-{MM}-{dd} {hh}:{mm}:{ss} {SSS}'` and it may be changed with `formatter`.

### Are there any API changes?

- [ ] The API has been changed.

### Related test cases or examples to use the new APIs

NA.

### Merging options

- [ ] Please squash the commits into a single one when merge.

### CODE REVIEW

File: src/util/format.ts [hunk 1]
```diff
@@ -93,21 +94,30 @@ export function formatTpl(
         return '';
     }
 
-    const $vars = paramsList[0].$vars || [];
-    for (let i = 0; i < $vars.length; i++) {
-        const alias = TPL_VAR_ALIAS[i];
-        tpl = tpl.replace(wrapVar(alias), wrapVar(alias, 0));
+    const isTimeAxis = paramsList[0].axisType.indexOf('time') >= 0;
```
Comment (susiwen8) [inline src/util/format.ts:97]: @Ovilia `axisType` might be `undefined
Screen Shot 2020-07-29 at 14 56 31
`

Comment (susiwen8) [reply src/util/format.ts:97]: For some charts which don't have `axis` like `tree` `treemap` `graph`

Comment (pissang) [reply src/util/format.ts:97]: Also found this issue.

Comment (pissang) [reply src/util/format.ts:97]: Fixed in https://github.com/apache/incubator-echarts/commit/353fee9d8d85be4727a95efbd021e82d1a77a060

Comment (plainheart) [reply src/util/format.ts:97]: In the latest commit, I only saw the change of css style, is there anything forgotten commit?

Comment (pissang) [reply src/util/format.ts:97]: Unfortunately yes. Committed the missded file.

File: src/util/time.ts [hunk 2]
```diff
@@ -224,3 +224,21 @@ export function getUnitValue (
...
+    if (!date || date instanceof Date) {
+        return date as Date;
+    }
+    if (date.indexOf(':') < 0) {
+        return new Date(date + ' 00:00:00');
+    }
+    else {
+        return new Date(date);
+    }
+}
```
[the remark this answers is not in the document]
Comment (plainheart) [reply src/util/time.ts:244]: Here is the same issue. https://github.com/apache/incubator-echarts/blob/2a909d9cfdf43714b1cfe90bd46cf48e6945ac79/src/util/time.ts#L147

**Pull request merged**